Description: Fix configure.in for autoreconf
 autoreconf complains about AC_DEFINE with only one argument, fix those.
Author: Carsten Leonhardt <leo@debian.org>
Last-Update: 2019-02-28
--- a/configure.in
+++ b/configure.in
@@ -10,10 +10,10 @@
 AC_PREFIX_DEFAULT(/usr/local)
 
 case "$host_os" in
-  *linux*) AC_DEFINE(LINUX)
+  *linux*) AC_DEFINE([LINUX], [1], [Linux])
 	TARGET=linux
     ;;
-  *solaris*) AC_DEFINE(SOLARIS)
+  *solaris*) AC_DEFINE([SOLARIS], [1], [Solaris])
 	TARGET=solarissparc
 	;;
   *sunos*)  TARGET=solarissparc
@@ -28,7 +28,7 @@
      ;;
   *HP*) TARGET=hpux
      ;;
-  *sequent*) AC_DEFINE(SEQUENT) 
+  *sequent*) AC_DEFINE([SEQUENT], [1], [Sequent])
 	;;
   *MINGW*) TARGET=mingw
      ;;
